.travis.yml

```diff
@@ -1,13 +1,18 @@
 ---
+# http://stackoverflow.com/questions/36979283/how-to-fix-travis-error-nomethoderror-undefined-method-spec-for-nilnilclass
+# https://github.com/travis-ci/travis-ci/issues/3531
+before_install:
+  - gem install bundler
+
 script: bundle exec rake
 
 rvm:
   - 1.9.3
   - 2.0
   - 2.1
-  - 2.2
+  - 2.2.5
   - rbx-2
-  - jruby
+  - jruby-9.0.5.0
```

Comment:
Before, this was picking jruby-1.7.something.

I admit that the Travis/RVM versioning logic is a mystery to me. In a project of mine where I try to use multiple rubies ([ruby-dbus](https://travis-ci.org/mvidner/ruby-dbus)) I care about the result being green for correct code, so I simply marked the less stable rubies that I don't care as much about as allowed failures:
```yml
matrix:
  allow_failures:
    - rvm: ruby-head
    - rvm: rbx
```